src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js

```diff
@@ -71,8 +71,7 @@ const FishBeltTransectInputs = ({
   const fishbelt_transect = validationsApiData?.fishbelt_transect
   // account for empty starter row
   const hasFishBeltObservations =
-    !!observationsState.length > 0 &&
-    Object.prototype.hasOwnProperty.call(observationsState[0], 'fish_attribute')
+    !!observationsState.length > 0 && observationsState[0]?.fish_attribute
```

Comment:
I really dont like this, but lets move on. If this gives us bugs, I think we need to push removing or changing the M804 feature

Reply:
agreed, it won't work if another row is missing a `fish_attribute` but the fix is complex . can come back to it later if it gives issues